src/popup/router/pages/SuccessTip.vue

```diff
@@ -42,38 +41,35 @@
     </div>
     <p
       v-if="!(tipUrl && isVerifiedUrl)"
-      class="f-14 sub-heading text-left"
+      class="sub-heading"
     >
       {{ $t('pages.successTip.note') }}
     </p>
-    <p class="f-18 my-35">
+    <p class="exclamation">
```

Comment:
Maybe `<em>` can suit this case and we can omit this class

Reply:
If we will use `<em>` for this, then we have to also define `margin` and `font-style`.